### PR TITLE
Docs updating eks command outputs

### PR DESCRIPTION
**Pull request by gflorinskaya**
http://docs-d2iq-com-pr-4029.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/index.md [hunk 1]
```diff
@@ -111,7 +109,134 @@ Before you begin, you must:
...
+[capi_book]: https://cluster-api.sigs.k8s.io/
+[calico]: https://docs.projectcalico.org/
+[capi]: https://github.com/kubernetes-sigs/cluster-api/tree/v0.3.20/
+[kcp]: https://github.com/kubernetes-sigs/cluster-api/tree/v0.3.20/controlplane/kubeadm
+[cabpk]: https://github.com/kubernetes-sigs/cluster-api/tree/v0.3.20/bootstrap/kubeadm
+[clusterresourceset_caep]: https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/proposals/20200220-cluster-resource-set.md
+[prerequisites]: ../prerequisites
+=======
+---
+layout: layout.pug
```
Comment (cmastr) [inline pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/index.md:131]: We can't have this header in the middle of  a topic like this.

File: pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/index.md [hunk 2]
```diff
@@ -111,7 +109,134 @@ Before you begin, you must:
...
+navigationTitle: Bootstrap
+title: Bootstrap
+menuWeight: 15
+excerpt: Prepare to deploy Kubernetes clusters
+enterprise: false
+---
+
+To create Kubernetes clusters, Konvoy uses [Cluster API][capi_book] (CAPI) controllers. These controllers run on a Kubernetes cluster. To get started, you need a _bootstrap_ cluster. By default, Konvoy creates a bootstrap cluster for you in a Docker container using the Kubernetes-in-Docker ([KIND][kind]) tool.
```
Comment (cmastr) [inline pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/index.md:139]: Where did this information come from? I'm not sure what we're doing here.

**Review by cmastr — CHANGES_REQUESTED** [review]
I think we need more work on the bootstrap section. Seems it got a little jumbled up.

File: pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/index.md (hunk 1 above)
Comment (gflorinskaya) [reply pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/index.md]: I accidentally broke this page, I hope I fixed it

File: pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/index.md [hunk 3]
```diff
@@ -27,24 +27,24 @@ Before you begin, you must:
...
-    INFO[2021-08-25T13:44:40-07:00] Initializing Cluster Autoscaler CustomResourceSet  src="bootstrap/clusterresourceset.go:181"
-    INFO[2021-08-25T13:44:40-07:00] Created Cluster Autoscaler CustomResourceSet  src="bootstrap/clusterresourceset.go:186"
-    INFO[2021-08-25T13:44:40-07:00] Initializing Node Feature Discovery CustomResourceSet  src="bootstrap/clusterresourceset.go:239"
-    INFO[2021-08-25T13:44:40-07:00] Created Node Feature Discovery CustomResourceSet  src="bootstrap/clusterresourceset.go:244"
-    INFO[2021-08-25T13:44:40-07:00] Initializing NVIDIA GPU Feature Discovery CustomResourceSet  src="bootstrap/clusterresourceset.go:297"
-    INFO[2021-08-25T13:44:40-07:00] Created NVIDIA GPU Feature Discovery CustomResourceSet  src="bootstrap/clusterresourceset.go:302"
+    INFO[2022-01-11T16:54:05-05:00] Creating bootstrap cluster                    src="bootstrap/bootstrap.go:148"
+    INFO[2022-01-11T16:54:09-05:00] Initializing bootstrap controllers            src="bootstrap/controllers.go:99"
+    INFO[2022-01-11T16:54:09-05:00] CAPI provider already exists!                 src="bootstrap/controllers.go:107"
```
Comment (msdolbey) [inline pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/index.md:32]: This probably makes NO difference at all... 

```suggestion
    INFO[2022-01-11T16:54:09-05:00] CAPI provider already exists!                 src="bootstrap/controllers.go:107"
```

File: pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/explore/index.md [hunk 4]
```diff
@@ -30,9 +30,12 @@ Before you start, make sure you have created a workload cluster, as described in
...
     ```sh
-    NAME                                         STATUS   ROLES                  AGE    VERSION
-    ip-10-0-126-209.us-west-2.compute.internal   Ready    control-plane,master   124m   v1.21.3
-    ip-10-0-204-168.us-west-2.compute.internal   Ready    <none>                 118m   v1.21.3
+    NAME                                         STATUS   ROLES    AGE   VERSION
+    ip-10-0-115-179.us-west-2.compute.internal   Ready    <none>   13m   v1.21.5-eks-bc4871b
+    ip-10-0-117-5.us-west-2.compute.internal     Ready    <none>   12m   v1.21.5-eks-bc4871b
+    ip-10-0-81-221.us-west-2.compute.internal    Ready    <none>   12m   v1.21.5-eks-bc4871b
+    ip-10-0-94-48.us-west-2.compute.internal     Ready    <none>   12m   v1.21.5-eks-bc4871b
+
```
Comment (msdolbey) [inline pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/explore/index.md:38]:
```suggestion
```

**Comment by msdolbey** [comment]
That's a really good point about the making EKS self-managed not being supported - @gflorinskaya - I'll make a PR against this feature branch ... and just yank that topic out - seems unnecessary

**Comment by msdolbey** [comment]
blah ... nm, I guess there's just enough info there - ANYWAY.


Great finds, @gflorinskaya

File: pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/index.md (hunk 2 above)
Comment (cmastr) [reply pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/index.md]: This info isn't in the published docs, so it slipped into your PR somehow. Can we fix it up?

https://docs.d2iq.com/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/

File: pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/index.md (hunk 1 above)
Comment (cmastr) [reply pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/index.md]: this looks good

File: pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/index.md (hunk 2 above)
Comment (cmastr) [reply pages/dkp/konvoy/2.1/choose-infrastructure/eks/advanced/bootstrap/index.md]: this is ok in the raw file.

**Review by cmastr — APPROVED** [review]

**Review by cneth — APPROVED** [review]

**Review by Colin1841 — APPROVED** [review]

**Comment by msdolbey** [comment]
Rebased.

**Pull request merged**